### ISSUE TITLE
Error ConjureCljSessionCloseAll

### ISSUE THREAD

**Issue by d2weber** [Olical/conjure#261]
When I create a new project with lein, start a `lein repl`, connect to it from Conjure and run `<leader>sQ` i get an error. In the repl log appears `Unknown session, correcting` and in the vim command output is shown 
```
Error executing vim.schedule lua callback: ...dweber/.config/nvim/plugged/conjure/lua/conjure/uuid.lua:26: bad argument #1 to 'sub' (string expected, got nil)
```
Those errors are shown continuously and don't recover and the only way out is to quit nvim forcefully.

**Comment by Olical** [comment]
So I still don't understand where your other session is coming from... are you running any nREPL tools that are connecting to the same server?

Because the behaviour I'm seeing here (which I can't reproduce locally!) is that when Conjure connects the REPL says "yep, I have a session for you", Conjure then tries to use that session and the REPL responds with "ERROR: Unknown session!" despite it just saying it existed!

So the new fix from my end is to say "okay, fair enough, I'll make another one then" and that one _does_ seem to work. So I don't think I can get rid of your two sessions, as far as I can tell this isn't happening anywhere else and I can't reproduce it, so my best solution for you is the one I've implemented: Creating a new session that _does_ work and ignoring the broken one.

I'm sorry that I can't explain why you have two and one of them is acting weird 😞 all I can do is mitigate it from Conjure's side. Is it working for you now though, despite the weird two sessions?

**Comment by d2weber** [comment]
> are you running any nREPL tools that are connecting to the same server?

I'm not sure what you mean. I guess no. My project.clj does contain only the necessary things and the ~/.lein directory does not contain configuration..

Now that I did some more experiments there seem to be always two sessions. No matter if I use your fix or the older of Conjure (fc07bf1a6353daa68b204d6301074200a7e7450d) and regardless of the lein version. My current workflow is:
- run `lein repl`
- start neovim which connects to the repl

Maybe it's just the other session started by the `lein repl` command? I'm new to Lein and Clojure so I'm not sure what I should expect.
Anyway, even if it was some kind of bug, that there are two sessions, it wouldn't be a problem for me. I was just mentioning it out of curiosity. And the initial issue seems to be fixed by your changes :+1:

**Comment by Olical** [comment]
I still don't understand the root cause but I'm glad the patch is helping! Closing this since it seems to have mitigated the issue. Thanks!
